.github/workflows/unit-tests.yaml

```diff
@@ -0,0 +1,25 @@
+name: Run Unit Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '18'
+
+      - name: Install dependencies
+        run: npm install && npm install @ironfish/rust-nodejs-linux-x64-gnu
```

Comment:
Why do you need that second install? Shouldn't that get installed with ironfish?

Reply:
It _should_ but it wasn't working properly. I posted about it here => https://iflabs.slack.com/archives/C02PF56HESK/p1691440865202389